fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h

```diff
@@ -1201,27 +1212,56 @@ class EngineControl {
       simN1 = simVars->getN1(engine);
       simN2 = simVars->getN2(engine);
       thrust = simVars->getThrust(engine);
+      engineFuelValveOpen = simVars->getValve(engine);
+      engineStarterPressurized = simVars->getStarterPressurized(engine);
+
+      // simulates delay to start valve open through fuel valve travel time
+      bool engineMasterTurnedOn = prevEngineMasterPos[engine - 1] < 1 && engineFuelValveOpen >= 1;
+
+      bool engineMasterTurnedOff = prevEngineMasterPos[engine - 1] == 1 && engineFuelValveOpen < 1;
 
-      // Set & Check Engine Status for this Cycle
-      engineStateMachine(engine, engineIgniter, engineStarter, simN2, idleN2, pressAltitude, ambientTemp,
-                         animationDeltaTime - prevAnimationDeltaTime);
       if (engine == 1) {
-        engineState = simVars->getEngine1State();
         deltaN2 = simN2 - simN2LeftPre;
         simN2LeftPre = simN2;
         timer = simVars->getEngine1Timer();
+        fbwN2 = simVars->getEngine1N2();
       } else {
-        engineState = simVars->getEngine2State();
         deltaN2 = simN2 - simN2RightPre;
         simN2RightPre = simN2;
         timer = simVars->getEngine2Timer();
+        fbwN2 = simVars->getEngine2N2();
+      }
+
+      // starts engines if Engine Master is turned on and Starter is pressurized or engine is still spinning fast enough
+      if (!engineStarter && engineFuelValveOpen == 1 && (engineStarterPressurized || simN2 >= 20)) {
```

Comment:
Will this work with out recommended binding for e.g. the TCA throttle? https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/?h=tca#cant-start-engines-when-using-tca-throttle

I can test it next week, but just want to make sure already.

Reply:
Looks good from my end.